src/app/scripts/cac/control/cac-control-sidebar-directions.js

```diff
@@ -300,11 +299,38 @@ CAC.Control.SidebarDirections = (function (_, $, Control, BikeModeOptions, Geoco
         }
     }
 
+    function reverseOriginDestination() {
+        // read what they are now
+        var origin = UserPreferences.getPreference('origin');
+        var originText = UserPreferences.getPreference('originText');
+        var destination = UserPreferences.getPreference('destination');
+        var destinationText = UserPreferences.getPreference('destinationText');
+
+        // update local storage
+        UserPreferences.setPreference('origin', destination);
```

Comment:
Curious as to what this does, as it looks at first glance that it wouldn't do anything since we just read these same values above and haven't modified them?